docs/releases/v2018.2.4.rst

```diff
@@ -0,0 +1,47 @@
+Gluon 2018.2.4
+==============
+
+Bugfixes
+~~~~~~~~
+
+* Fixes device alias for Ubiquiti UniFi AC LR. (`#1834 <https://github.com/freifunk-gluon/gluon/issues/1834>`_)
+
+* Uses correct ath10k firmware for OCEDO Koala. (`#1838 <https://github.com/freifunk-gluon/gluon/pull/1838>`_)
+
+* Fixes some batman-adv bugs with backports from 2019.4 and 2019.5 by updating the openwrt-routing packages feed
+
+Other changes
+~~~~~~~~~~~~~
+
+* Linux kernel has been updated to either
+  
+  - 4.9.207 (ar71xx, brcm2708, mpc85xx) or
```

Comment:
Would drop the "or" in this listing

Reply:
was copied from v2018.2.3 and is necessary because of the "either" in the sentence above, isn't it?

Reply:
I would drop the either also then. But this is more of a personal preference.